src/index.js

```diff
@@ -179,6 +181,30 @@ class DAPIClient {
    */
   getHistoricBlockchainDataSyncStatus() { return this.makeRequestToRandomDAPINode('getHistoricBlockchainDataSyncStatus', {}); }
 
+  /**
+   * Retrieve user's last state transition hash
+   * @param {string} userId
+   * @returns {Promise<string>}
+   */
+  async getLastUserStateTransitionHash(userId) {
+    const request = new LastUserStateTransitionHashRequest();
+    request.setUserId(userId);
+
+    const nodeToConnect = await this.MNDiscovery.getRandomMasternode();
+
+    const client = new CorePromiseClient(`${nodeToConnect.getIp()}:${this.getGrpcPort()}`);
+
+    const response = await client.getLastUserStateTransitionHash(request);
+
+    const hashBuffer = response.getStateTransitionHash();
+
+    if (Buffer.isBuffer(hashBuffer)) {
+      return hashBuffer.toString('hex');
+    }
+
+    return hashBuffer;
```

Comment:
I think it will be an empty Buffer, isn't it?

Reply:
I would suggest to test it when the server will be ready.

Reply:
@shumkov It turns out, in case if we submit `null` to `setStateTransitionHash` function of a response class. We get an empty string as a result of calling `getStateTransitionHash`. We need to submit an empty buffer in case we want to get an empty buffer.

Reply:
In this case, if a buffer is empty we need to respond with null to users, right?